sdk/src/client.ts

```diff
@@ -94,10 +104,7 @@ export class VaultClient {
         wallet: anchor.Wallet,
         env: DeploymentEnv,
         reserveTokenMint: PublicKey,
-        solend: SolendReserveAsset,
-        port: PortReserveAsset,
-        jet: JetReserveAsset,
-        owner: PublicKey,
+        owner: Keypair,
```

Comment:
Why does this need to be a Keypair?

Reply:
Also applies to L226 et al

Reply:
Technically line 107 doesn't have to be keypair.
Later, it should be keypair because IMO only the owner can authorize yield src initialization, so the owner's signature is needed.